### PR TITLE
Home: Highlight download links with a small border

### PR DESCRIPTION
**Pull request by hoffie**
**Short description of changes**

This adjusts the styling of the header's download links to
- add a light bottom border for non-hovered links to make it obvious that they are clickable
- remove the text-underlining on hover as it would lead to two lines
- highlight the border on hover

**Context: Fixes an issue? Related issues**

It's hard to see that the the platform names are clickable. I'm open to other suggestions, but this sounded like the most-straightfoward solution for now.

**Status of this Pull Request**

Ready.

**What is missing until this pull request can be merged?**

Reviews.

**Does this need translation?**
No.

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch: pretty much... :)

### CODE REVIEW

**Comment by ann0see** [comment]
Initially they were kind of hidden due to the reason that we wanted people to first read the specific install guides (Windows needs ASIO etc). I think they're ok, but still dark mode looks a bit strange (contrast?)

grafik

**Comment by hoffie** [comment]
> Initially they were kind of hidden due to the reason that we wanted people to first read the specific install guides (Windows needs ASIO etc).

Understood, but we also have to consider existing users which are only looking for the latest version for updating.

> I think they're ok, but still dark mode looks a bit strange (contrast?)

I've added a rule for dark mode to match the regular link color. Not sure if I like it, but this way it's consistent at least and I don't want to introduce yet another color. :)

I've also rebased.

File: assets/css/home.css [hunk 1]
```diff
@@ -32,28 +32,32 @@ header {
...
+    color: #dbdbdb;
+    border-color: #dbdbdb;
+  }
+
+  .mainsite header h1, .mainsite, header a {
+    color: #dbdbdb;
+  }
+  #quick_dl_container a {
+    border-bottom-color: #3daed2;
+  }
```
Comment (hoffie) [inline assets/css/home.css:60]: Had to move this block around in order for it to match.

**Review by ann0see — APPROVED** [review]
Ok. Tested and I think it's ok like that.

**Pull request merged**